dags/prio/processor.py

```diff
@@ -0,0 +1,276 @@
+    python_callable=clean_buckets,
+    op_kwargs={
+        "private_bucket": "project-a-private",
+        "shared_bucket": "project-a-shared",
+        "google_cloud_storage_conn_id": "google_cloud_prio_a",
+    },
+    dag=dag,
+)
+
+clean_processor_b = PythonOperator(
+    task_id="clean_processor_b",
+    python_callable=clean_buckets,
+    op_kwargs={
+        "private_bucket": "project-b-private",
+        "shared_bucket": "project-b-shared",
+        "google_cloud_storage_conn_id": "google_cloud_prio_b",
+    },
+    dag=dag,
+)
+
+load_processor_a = GoogleCloudStorageToGoogleCloudStorageOperator(
+    task_id="load_processor_a",
+    source_bucket="moz-fx-data-prio-data",
+    source_object="staging/submission_date={{ ds }}/server_id=a/*",
+    destination_bucket="project-a-private",
+    destination_object="raw/submission_date={{ ds }}/",
+    google_cloud_storage_conn_id="google_cloud_prio_admin",
+    dag=dag,
+)
+
+load_processor_b = GoogleCloudStorageToGoogleCloudStorageOperator(
+    task_id="load_processor_b",
+    source_bucket="moz-fx-data-prio-data",
+    source_object="staging/submission_date={{ ds }}/server_id=b/*",
+    destination_bucket="project-b-private",
+    destination_object="raw/submission_date={{ ds }}/",
+    google_cloud_storage_conn_id="google_cloud_prio_admin",
+    dag=dag,
+)
+
+trigger_processor_a = GoogleCloudStorageToGoogleCloudStorageOperator(
+    task_id="trigger_processor_a",
+    source_bucket="moz-fx-data-prio-data",
+    source_object="staging/_SUCCESS",
+    destination_bucket="project-a-private",
+    destination_object="raw/_SUCCESS",
+    google_cloud_storage_conn_id="google_cloud_prio_admin",
+    dag=dag,
+)
+
+trigger_processor_b = GoogleCloudStorageToGoogleCloudStorageOperator(
+    task_id="trigger_processor_b",
+    source_bucket="moz-fx-data-prio-data",
+    source_object="staging/_SUCCESS",
+    destination_bucket="project-b-private",
+    destination_object="raw/_SUCCESS",
+    google_cloud_storage_conn_id="google_cloud_prio_admin",
+    dag=dag,
+)
+
+# Initiate the processors by spinning up GKE clusters and running the main
+# processing loop. These SubDags are coordinated by a higher level graph, but
+# they can be decoupled and instead run on the same schedule. This structure is
+# done for convenience, since both processors are running on the same
+# infrastructure (but partitioned along project boundaries).
+
+processor_a = SubDagOperator(
+    subdag=processing.prio_processor_subdag(
+        parent_dag_name=dag.dag_id,
+        child_dag_name="processor_a",
+        default_args=DEFAULT_ARGS,
+        server_id="a",
+        gcp_conn_id="google_cloud_prio_a",
+        service_account="prio-runner-a@moz-fx-priotest-project-a.iam.gserviceaccount.com",
+        arguments=["processor/bin/process"],
+        env_vars={
+            "DATA_CONFIG": "/app/processor/config/content.json",
+            "SERVER_ID": "A",
+            "SHARED_SECRET": "m/AqDal/ZSA9597GwMM+VA==",
```

Comment:
Should this be here?

Reply:
Good point, this should probably be part of the airflow variables too.